string latRMC
string dirLatRMC
string lonRMC
string dirLonRMC
float64 speedmsRMC
string latGGA
string dirLatGGA
string lonGGA
string dirLonGGA
string GPSqualityGGA
string hnnGGA
